src/content/en/2020/markup.md

```diff
@@ -626,34 +704,42 @@ Using `target="_blank"` is known as a [security vulnerability](https://mathiasby
+Yet what we observe with this year's data is that there's a lot of movement at the bottom and near the shore of said sea of HTML. We approach near-complete adoption of living HTML, are quick to prune our pages of fads like Google and Facebook widgets, and we're fast in adopting and shunning frameworks. Both Angular and AMP, though "component frameworks", seem to have significantly lost in popularity, likely for solutions like React and Vue.
 
-That suggests us to close with an appeal:
+And still, there are no signs we exhausted the options HTML gives us. The median of 30 different elements used on a given page, which is roughly a quarter of the elements HTML provides us with, suggests a rather one-sided use of HTML. That is supported by the immense popularity of elements like `div` and `span`, and no custom elements to potentially meet the demands that these two elements may represent. Unfortunately, we couldn't validate each document in the sample. However, anecdotally and to be taken with caution, we learned that [79%](https://github.com/HTTPArchive/almanac.httparchive.org/issues/899#issuecomment-717856201) of W3C-tested documents have validation errors. After everything we've seen, it looks like we're still far from mastering the craft of HTML.
 
-Pay attention to HTML. Focus on HTML. It’s important and worthwhile to invest in HTML. HTML is a document language that may not have the charm of a programming language, and yet the Web is built on it. Use less HTML—learn what’s really needed. Use more appropriate HTML—learn what’s available and what it’s there for. And [validate](https://validator.w3.org/docs/why.html) your HTML. Everyone can write invalid HTML—just invite the next person you meet to write an HTML document and validate the output—, but a professional developer can be expected to produce valid HTML. Writing correct and valid HTML is a craft to take pride in.
+That compels us to close with an appeal. Pay attention to HTML. Focus on HTML. It's important and worthwhile to invest in HTML. HTML is a document language that may not have the charm of a programming language, and yet the web is built on it. Use less HTML and learn what's really needed. Use more appropriate HTML—learn what's available and what it's there for. And [validate](https://validator.w3.org/docs/why.html) your HTML. Anyone can write invalid HTML (just invite the next person you meet to write an HTML document and validate the output) but a professional developer can be expected to produce valid HTML. Writing correct and valid HTML is a craft to take pride in.
```

Comment:
This happens throughout the doc, but here I find the change from uppercase “Web” to lowercase “web” most dramatic. Which web is meant here? Yes, probably people guess which one. But _the Web_ is _the Web_, is a statement, is an exclamation mark. Here, at last, the effort to be consistently lowercase reduces the meaning and weight, perhaps of the whole chapter. (I added a bit of drama, on purpose, but really—HTML makes for the _Web_.)

Reply:
This is a conscious choice in our style guide: https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide#user-content-technical:~:text=Use%20lowercase%20%22web%22

Do you feel strongly that this usage is significant enough compared to lowercase phrases like "web page" or "web developer" to break that colloquialism?

Reply:
Personally, yes, very much! I think there’s the Web, one Web, and then many websites, web apps, web developers, &c. (I never got why one would write “web” lowercase—I think “which web?” then.) But that’s just me; to make the point I think at least here it should be “Web”; and if that is a problem and we need to game the style guide, then maybe “World Wide Web”? But there I’m not sure we need to go there.

Reply:
I had the same issue when writing my book I used Web and Internet but publishers insisted on web and internet. I’ve come round to lowercase and prefer it now but it took a while to persuade me. Apparently lowercase is more and common and the capitalisation is seen as somewhat old fashioned nowadays in style guides: https://www.wired.com/2016/04/ap-finally-realizes-2016-will-let-us-stop-capitalizing-internet/

Reply:
Given that we use "web" lowercase even outside of pairings like "web page" and "web developer", I'm inclined to keep this consistent. For example, the home page prominently refers to this as "The 2020 state of the web report". It'd be strangely inconsistent to make it a proper noun in this one instance. Regarding "gaming" the style guide, we _explicitly discourage_ "World Wide Web"! 😄 

This is more of a question about overall tone and formality, which goes beyond just this chapter, similar to the quotes thing. Since both Web and web are accepted and we've standardized on the latter, I think we should stick to it for now and reevaluate later if necessary.

PS I got a laugh out of this quote from the Wired article:

> There is no earthly reason to capitalize any of these words. Actually there never was.

Reply:
I think we should resolve this. We need consistency between all the chapters and we've decided to use lowercase form for the Web Almanac. I get that others (including the authors here) may prefer the other way, but we need the consistency so that isn't something we can resolve here.

@j9t please raise a separate issue for discussion and visibility to the rest of the contributors if you still feel that strongly about this (like I did for smart quotes). Also means we can refer to it, if this item comes up again in another chapter - especially if we as a team decide to change to capitalised form.